Decred Wallet/Custom Views/AccountTableView.swift

```diff
@@ -0,0 +1,168 @@
+        line.backgroundColor = .init(red: 0.9, green: 0.92, blue: 0.93, alpha: 1)
+        self.bgView?.addSubview(line)
+        
+        self.tableView = UITableView.init(frame: CGRect(x: 0, y: 50, width: frame.width, height: (self.bgView?.frame.height)!-50), style: .grouped)
+        self.tableView?.backgroundColor = .white
+        self.tableView?.delegate = self as UITableViewDelegate
+        self.tableView?.dataSource = self as UITableViewDataSource
+        self.bgView?.addSubview(self.tableView!)
+        self.tableView?.separatorStyle = .none
+        self.tableView?.register(UINib.init(nibName: "AccountTableViewCell", bundle: nil), forCellReuseIdentifier: "AccountTableViewCell")
+        
+        self.showView()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    func setAccount() {
+        let acc = AppDelegate.walletLoader.wallet?.walletAccounts(confirmations: 0)
+        
+        let defaultAccount = acc?.filter({ $0.isDefault}).first
+        if (defaultAccount != nil) {
+            let defaultAcc: [DcrlibwalletAccount] = [defaultAccount!]
+            self.listData?.append(defaultAcc)
+            
+            let walletAccountsArr: [DcrlibwalletAccount] = (acc?.filter({!$0.isHidden && $0.number != INT_MAX }))!
+            self.listData?.append(walletAccountsArr)
+        }else{
+           let walletAccountsArr: [DcrlibwalletAccount] = (acc?.filter({!$0.isHidden && $0.number != INT_MAX }))!
+           self.listData?.append(walletAccountsArr)
+        }
+        
+        self.tableView?.reloadData()
+    }
+    
+    //MARK: tableViewDelegate
+    func numberOfSections(in tableView: UITableView) -> Int {
+        return self.listData!.count
+    }
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        let walletAccountsArr: [DcrlibwalletAccount]? = self.listData![section]
+        
+        return walletAccountsArr!.count
+    }
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 78
+    }
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 30
+    }
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        return 0.1
+    }
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell:AccountTableViewCell = tableView.dequeueReusableCell(withIdentifier: "AccountTableViewCell", for: indexPath) as! AccountTableViewCell
+        
+        let cellBgView: UIView? = UIView.init()
+        cellBgView?.backgroundColor = .white
+        cell.selectedBackgroundView = cellBgView
+        
+        let walletAccountsArr: [DcrlibwalletAccount] = (self.listData?[indexPath.section])!
+        let  walletAccount = walletAccountsArr[indexPath.row]
+        
+        cell.setAccount(walletAccount)
```

Comment:
suggestion for line 120-123:
`cell.setAccount(self.listData[indexPath.section][indexPath.row])`